twitoff/twitter.py

```diff
@@ -33,7 +33,7 @@ def add_or_update_user(username):
 
         for tweet in tweets:
             vectorized_tweet = vectorize_tweet(tweet.full_text)
-            db_tweet = Tweet(id=tweet.id, text=tweet.full_text,
+            db_tweet = Tweet(id=tweet.id, text=tweet.full_text[0:300],
```

Comment:
If after deploying you encounter errors when trying to update a user who has a tweet over 300 chars: either change the database model to allow for more characters and remember to re-migrate, or just choose to store the first 300 characters of the tweet text (as shown here).